packages/webapp-libs/webapp-core/src/components/buttons/link/link.component.tsx

```diff
@@ -19,20 +22,29 @@ export type ExternalLinkProps = AnchorHTMLAttributes<HTMLAnchorElement> & Button
 export type LinkProps = InternalLinkProps | InternalNavLinkProps | ExternalLinkProps;
 
 export const Link = (props: LinkProps) => {
-  const { variant = ButtonVariant.LINK, size = ButtonSize.NORMAL, children, icon, ...linkProps } = props;
+  const { variant = ButtonVariant.LINK, size = ButtonSize.NORMAL, children, icon, className, ...linkProps } = props;
+
+  const renderIcon = () => icon && <span className="mr-1">{icon}</span>;
```

Comment:
What do you think about moving this method to `button.utils.tsx` and use it here and in `<Button/>` component?